api/proto/teleport/legacy/client/proto/authservice.proto

```diff
@@ -1151,6 +1151,32 @@ message ListRolesResponse {
 
   // NextKey will serve as the StartKey for the next page of roles.
   string NextKey = 2;
+
+  // RequestableRoles is a page of requestable roles. This will be populated if the request only requested requestable roles.
+  repeated types.RequestableRole RequestableRoles = 3;
+}
```

Comment:
I believe this is a holdover from a previous iteration of this PR and can be removed.